controllers/akodeploymentconfig/akodeploymentconfig_controller_avi_phase.go

```diff
@@ -203,26 +204,26 @@ func (r *AKODeploymentConfigReconciler) reconcileNetworkSubnets(
 	return res, nil
 }
 
-// func (r *AKODeploymentConfigReconciler) reconcileCloudUsableNetwork(
-// 	ctx context.Context,
-// 	log logr.Logger,
-// 	obj *akoov1alpha1.AKODeploymentConfig,
-// ) (ctrl.Result, error) {
-// 	log = log.WithValues("cloud", obj.Spec.CloudName)
-// 	log.Info("Start reconciling AVI cloud usable network")
-
-// 	added, err := r.AddUsableNetwork(r.aviClient, obj.Spec.CloudName, obj.Spec.DataNetwork.Name)
-// 	if err != nil {
-// 		log.Error(err, "Failed to add usable network", "network", obj.Spec.DataNetwork.Name)
-// 		return ctrl.Result{}, err
-// 	}
-// 	if added {
-// 		log.Info("Added Usable Network", "network", obj.Spec.DataNetwork.Name)
-// 	} else {
-// 		log.Info("Network is already one of the cloud's usable network", "network", obj.Spec.DataNetwork.Name)
-// 	}
-// 	return ctrl.Result{}, nil
-// }
+func (r *AKODeploymentConfigReconciler) reconcileCloudUsableNetwork(
+	ctx context.Context,
+	log logr.Logger,
+	obj *akoov1alpha1.AKODeploymentConfig,
+) (ctrl.Result, error) {
+	log = log.WithValues("cloud", obj.Spec.CloudName)
+	log.Info("Start reconciling AVI cloud usable network")
+
+	if err := r.AddUsableNetwork(r.aviClient, obj.Spec.CloudName, obj.Spec.ControlPlaneNetwork.Name, log); err != nil {
+		log.Error(err, "Failed to add usable network", "network", obj.Spec.ControlPlaneNetwork.Name)
+		return ctrl.Result{}, err
```

Comment:
should this be a warning instead? if we aren't able to add usable network we don't want the whole reconcilation to be blocked as well

Reply:
@HanFa , I think that's a good point, could you please update it accordingly? Thanks!

Reply:
let's keep it for now unless we observe it erroring out in any other senarios